consensus/scp/src/node.rs

```diff
@@ -63,12 +62,22 @@ impl<V: Value, ValidationError: Display> Node<V, ValidationError> {
         Q: QuorumSet,
         validity_fn: ValidityFn<V, ValidationError>,
         combine_fn: CombineFn<V>,
+        current_slot_index: SlotIndex,
```

Comment:
should this always be zero?

Reply:
My intent is that Node gets instantiated to work on whatever the next slot is. For example, if the consensus service restarts it should start running consensus at the current slot index for is local ledger, instead of starting over at slot zero.